src/render/dom/index.ts

```diff
@@ -104,9 +105,13 @@ export const motion = /*@__PURE__*/ createMotionProxy(allMotionFeatures)
  *
  * @public
  */
-export function createDomMotionComponent(key: string) {
+export function createDomMotionComponent(key: HTMLElements) {
```

Comment:
This should remain `string` as it can be used to create custom HTML elements.

Reply:
Should I keep string only or just merging types like `HTMLElements | string` to give user some hints?

Reply:
Er yeah you could merge and add SVGElements too